js/package.json

```diff
@@ -54,7 +54,7 @@
   },
   "jupyterlab": {
     "extension": "src/jupyterlab-plugin",
-    "outputDir": "../share/jupyter/labextensions/jupyter-threejs",
+    "outputDir": "../pythreejs/labextension",
```

Comment:
This doesn't match the `"share/jupyter/labextensions/jupyter-threejs"` in setup.py right?

Reply:
Ah, looks like some of my changes have mixed up... I'll resolve this

Reply:
I think this is correct: the outputdir maps to `<ROOT>/pythreejs/labextension`, and `setup.py` looks for `<ROOT>/pythreejs/labextension`